src/controller/java/AndroidCallbacks.cpp

```diff
@@ -365,23 +365,9 @@ void ReportCallback::OnEventData(const app::EventHeader & aEventHeader, TLV::TLV
     readerForJson.Init(*apData);
 
     jlong eventNumber   = static_cast<jlong>(aEventHeader.mEventNumber);
-    jlong priorityLevel = static_cast<jint>(aEventHeader.mPriorityLevel);
+    jint priorityLevel = static_cast<jint>(aEventHeader.mPriorityLevel);
     jlong timestamp     = static_cast<jlong>(aEventHeader.mTimestamp.mValue);
 
-    jobject value = nullptr;
-#if USE_JAVA_TLV_ENCODE_DECODE
-    TLV::TLVReader readerForJavaObject;
```

Comment:
please do not skip it

Reply:
we would decouple it via feature flag, please do not remove this part of code right now

Reply:
Acked